rmj_oss/server.py

```diff
@@ -124,5 +122,21 @@ def searchPost(column, value):
     else:
         return "404-Page not Found" 
 
+# Given a post's id, checks for existence and then deletes post
+@app.route("/api/delete-post", methods=['POST'])
+def deletepost():
+    post_id = request.form["post_id"]
+
+    # Existence check
+    dne = session.query(RentPost).filter_by(id=post_id).scalar() is None
+    if dne:
+        return str('Error - Requested post ID does not exist.')
+
+    post_to_delete = session.query(RentPost).filter_by(id=post_id).one()
+    post_title = post_to_delete.title
+    session.delete(post_to_delete)
+    session.commit()
+    return "Deleted ID: " + post_id + ", TITLE: " + post_title
```

Comment:
We should define a rule defining if we should use " or ' for strings

Reply:
Agreed. We should bring this up in the next meeting.